lib/sapience.rb

```diff
@@ -1,5 +1,42 @@
 require "sapience/version"
+require "sapience/sapience"
 
+# @formatter:off
 module Sapience
-  # Your code goes here...
+  autoload :AnsiColors,         "sapience/ansi_colors"
```

Comment:
Is there a good reason for using autoload, it is on the whole not thread safe, and it is much easier to make stuff work reliably in puma if we just require everything up front.